frontend/src/pages/Report.tsx

```diff
@@ -655,25 +657,80 @@ export const Report = () => {
                   const dateStr = formatDate(date, dateFormat);
                   return (
                     <div key={dateStr} className="col-1 cell-container">
-                      <input
-                        aria-labelledby={`total of hours spent during the day ${dateStr}`}
-                        type="text"
-                        id={dateStr}
-                        className="cell not-outline"
-                        value={getTotalHours(dateStr)}
-                        readOnly
-                      />
+                      <div className="comment-container">
+                        <input
+                          aria-labelledby={`total of hours spent during the day ${dateStr}`}
+                          type="text"
+                          id={dateStr}
+                          className="cell not-outline"
+                          value={getTotalHours(dateStr)}
+                          readOnly
+                        />
+                        {/* Only show warnings for weeks that have passed. 
+                          It must be at least Saturday. */}
+                        {new Date().getTime() - currentWeekArray[4].getTime() >
+                        1000 * 3600 * 24 ? (
```

Comment:
https://date-fns.org/v2.28.0/docs/isPast

Reply:
Perfect! I think that one should work. Will look into it and change it.